src/components/SearchPage/SearchResult/SearchResult.js

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import { Container, Row, Col, Button, Card } from 'react-bootstrap';
+import Avatar from './images/avatar.png';
+import './style.css';
+function SearchResult() {
+  return (
+    <div className="SearchResult">
+      <Container>
+        <Row>
+          <Col>
+            <h3>Search result</h3>
+          </Col>
+
+          <hr className="solid" />
+        </Row>
+        <Row>
+          <Col md={4}>
```

Comment:
Don't copy and paste yourself.

- make an array of objects (which each object represent that doctor info)
- then map over them.
this will make you write the Card only one time 

see this ex: https://codesandbox.io/s/compassionate-christian-945lt?file=/src/App.js